src/components/message/type-icon/type-icon-classic.style.js

```diff
@@ -0,0 +1,15 @@
+import { css } from 'styled-components';
+import { THEMES } from '../../../style/themes';
+
+export default ({
+  theme, transparent, messageType, roundedCorners
+}) => theme.name === THEMES.classic
+  && css`
+    border-radius: ${roundedCorners ? '3px 0 0 3px' : '0px'};
+    background-color: ${transparent ? theme.colors.white : theme.colors[messageType][0]};
+    span {
+      &:before {
+        color: ${transparent ? theme.colors[messageType][0] : theme.colors.white};
```

Comment:
We should probably avoid relying on index positions, I'm left wondering why theme.colors[whatever] is an array.